tests/unit/Application/Service/InformationServiceTest.php

```diff
@@ -48,6 +46,12 @@ public function setUp()
         $this->service = new InformationService($this->apiCollection, $logger);
     }
 
+    public function tearDown()
```

Comment:
FYI: You can use a trait, that would also update the PHPUnit assertion counter.

http://docs.mockery.io/en/latest/reference/phpunit_integration.html

Reply:
Fixed this! Thanks for the tip